infra/data_bucket.tf

```diff
@@ -54,13 +59,9 @@ locals {
     "user:stephen.compall@digitalasset.com",
     "user:victor.mueller@digitalasset.com",
   ]
-}
-
-resource "google_storage_bucket_iam_member" "appr" {
-  for_each = toset(local.appr_team)
-  bucket   = google_storage_bucket.data.name
-  role     = "roles/storage.objectViewer"
-  member   = each.key
+  canton_team = [
+    "user:oliver.seeliger@digitalasset.com",
```

Comment:
let’s make sure we extend this sooner rather than later and add the whole Canton team in there. But doesn’t haev to be in this PR.

Reply:
I'd like it to be in this PR.

Reply:
Listing the members of the canton team here, but keep in mind that potentially every engineer (and even likely a few non-engineers as well) will need the ability to build canton locally.

Canton members (alphabetical all `@digitalasset.com`):

- andreas.lochbihler
- arne.gebert
- danilo
- matthias.schmalz
- oliver.seeliger
- phoebe.nichols
- rafael.guglielmetti
- ratko
- soren.bleikertz

Other prior, "active" canton committers:

- brian.healey
- daniel.porter
- moritz.kiefer
- remy.haemmerle
- shaul
- simon.maxen
- tudor.voicu
- victor.mueller

Other engineers that might depend on building canton locally:
- gary.verhaegen
- support engineers?
- cx-engineering?
- other members of the kv teams?